src/m365/entra/commands/m365group/m365group-user-remove.ts

```diff
@@ -91,70 +128,38 @@ class EntraM365GroupUserRemoveCommand extends GraphCommand {
   }
 
   #initOptionSets(): void {
-    this.optionSets.push({ options: ['groupId', 'teamId'] });
+    this.optionSets.push(
+      {
+        options: ['groupId', 'teamId', 'groupName', 'teamName']
+      },
+      {
+        options: ['userName', 'ids', 'userNames']
+      }
+    );
+  }
+
+  #initTypes(): void {
+    this.types.string.push('groupId', 'groupName', 'teamId', 'teamName', 'userName', 'ids', 'userNames');
+    this.types.boolean.push('force');
   }
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
     await this.showDeprecationWarning(logger, aadCommands.M365GROUP_USER_REMOVE, commands.M365GROUP_USER_REMOVE);
```

Comment:
I think we are missing a depreacted info regarding the `userName`. 
We should add a notify that this should be avoided